### PR TITLE
bugfix/account-id

### PR DESCRIPTION
**Pull request by kosovych**

### CODE REVIEW

File: src/store/auth/operations/loginOperation.js [hunk 1]
```diff
@@ -25,14 +25,14 @@ export const loginOperation = createLogic({
       )
       const sessionRequestToken = sessionRequestTokenReq.data.request_token
 
-      const sessionIDReq = await axios.post(
+      const sessionIdReq = await axios.post(
```
Comment (VladWithCookies) [inline src/store/auth/operations/loginOperation.js:28]: sessionIdReq -> response

File: src/store/auth/reducer.js [hunk 2]
```diff
@@ -9,17 +9,18 @@ import {
...
   switch (action.type) {
     case LOGIN_REQUEST: return { ...state, loading: true, errorTxt: null }
     case LOGIN_FAILS: return { ...state, errorTxt: action.errorTxt, loading: false }
-    case LOGIN_SUCCESS: return { ...state, sessionID: action.sessionID, loading: false }
+    case LOGIN_SUCCESS: return { ...state, sessionId: action.sessionId, loading: false }
     case LOGIN_LOGOUT:
-      localStorage.removeItem('session_id')
-      return { ...state, sessionID: null }
+      localStorage.removeItem('sessionId')
+      localStorage.removeItem('userId')
```
Comment (VladWithCookies) [inline src/store/auth/reducer.js:22]: async operation should go to the logic

File: src/store/auth/selectors.js [hunk 3]
```diff
@@ -7,10 +7,35 @@ const userDataSelector = state => (
...
+)
+
+export const usernameSelector = createSelector(
+  userDataSelector,
+  userData => get(userData, 'username')
+)
+
+export const errorTxtSelector = createSelector(
+  userDataSelector,
+  userData => get(userData, 'errorTxt')
```
Comment (VladWithCookies) [inline src/store/auth/selectors.js:40]: don't use shortcuts for variable names

File: src/containers/Login/container.js [hunk 4]
```diff
@@ -26,9 +27,9 @@ Login.propTypes = {
 }
 
 const mapStateToProps = state => ({
-  loading: state.auth.loading,
-  errorTxt: state.auth.errorTxt,
-  sessionID: state.auth.sessionID
+  loading: authLoadingSelector(state),
+  errorTxt: errorTxtSelector(state),
```
Comment (VladWithCookies) [inline src/containers/Login/container.js:31]: don't use shortcuts for selector names

File: src/store/auth/operations/loginOperation.js (hunk 1 above)
Comment (kosovych) [reply src/store/auth/operations/loginOperation.js:28]: @VladWithCookies I can't call in just `response` because there is few requests

Comment (VladWithCookies) [reply src/store/auth/operations/loginOperation.js]: it any it is not request, but response )

**Review by VladWithCookies — APPROVED** [review]

**Pull request merged**